dataland-frontend/src/components/general/TheBrands.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <section v-if="brandsSection" class="brands" role="region" aria-label="Brands Statement">
+    <div class="brands__wrap">
+      <h2 id="brands-heading" aria-labelledby="brands-heading" class="brands__text">
+        {{ brandsSection.text[0] }}
+        <span>{{ brandsSection.text[1] }}</span>
+      </h2>
+      <div class="brands__list" role="list">
+        <div class="brands__item" v-for="(imgSrc, index) in brandsSection.image" :key="index" role="listitem">
+          <img :src="imgSrc" :alt="`Brand ${index + 1}`" :class="`brands__item-image brands__item-image--${index}`" />
```

Comment:
Why is the dynamic css class `brands__item-image--${index}` being assigned to the `<img>` ?
The images seem to have the same styling for each index when I visit the page.

Reply:
It was to handle the mouse over effect for different logos, but this feature will be added in the future. For now I removed it.